provision/acc_provision/templates/aks-provision-config.yaml

```diff
@@ -9,22 +9,30 @@ aci_config:
   tenant:
     name: testTenant
   vrf:                          # This VRF used to create all kubernetes EPs
-    name: vpcVrf                # This is the vrf attached to the CCP. Will be created
+    name: vnetVrf                # This is the vrf attached to the CCP. Will be created
                                 # if not present
     tenant: testTenant
-    region: region-xxx          # your aws region
+    region: region-xxx          # your azure region
   #custom_epgs:                 # List of additional endpoint-group names
   #  - epg-a                    # to configure for use with annotations
   #  - custom_group2
 
 #
-# Networks used by ACI containers/IPI
+# Networks used by ACI containers/AKS
 #
 net_config:
   pod_subnet: 10.14.0.1/16      # Subnet to use for Kubernetes pods
-  node_subnet: 10.0.0.1/24      # Subnet where AKS nodes will be deployed
+  machine_cidr: 10.0.0.0/16     # cidr for aks vmscaleset
+  node_subnet: 10.0.0.1/24      # must be part of machine_cidr
+  extern_dynamic: 10.3.0.1/24   # Subnet to use for dynamic external IPs
+  extern_static: 10.4.0.1/24    # Subnet to use for static external IPs
+  node_svc_subnet: 10.5.0.1/24  
   opflex_server_port: 19999 
   interface_mtu: 1450          # min = 1280 for ipv6, max = 8900 for VXLAN
+  #service_monitor_interval: 5  # IPSLA interval probe time for PBR tracking
+                                # default is 5, set to 0 to disable, max: 65535
+  #pbr_tracking_non_snat: true  # Default is false, set to true for IPSLA to
+                                # be effective with non-snat services
```

Comment:
Are these two relevant to AKS?

Reply:
They are not relevant. Even if user adds, it will be ignored. It's there only because of the common template.